### ISSUE TITLE
[Feature Request]Diffview for commit which modified current line

### ISSUE THREAD

**Issue by sakelkar** [sindrets/diffview.nvim#307]
Hi, 

Thanks for the great plugin. 

Will you please consider, augmenting DiffviewFileHistory or DiffviewOpen to show diffs from commit that modified current line under the current file?

Many times getting the full context of the commit when current line was modified is very very useful. Helps to get the context very quickly. Typically when one is working on hot fixes and not a big feature.    

Thanks
Sameer

**Comment by sindrets** [comment]
https://github.com/sindrets/diffview.nvim/blob/d539ef98e4809a38de3d9ea53697fd2f5ad337a6/doc/diffview.txt#L151-L153

**Comment by sakelkar** [comment]
I had tried that option and sure it traces of the given line. I used :DiffviewFileHistory -L<Line1>,<Line1>:<filename> and it only traces current file for that line.  

But I am looking to *cycle through* other changes which were part of *that* commit when current line changed last time. Since your plugin, is actually centered around *cycling through* files in a given commit, I thought something like this will also be useful. Apologies if I am missing something which is already present. 

Thanks,
Sameer
